### PR TITLE
Updated Masonry docs to have better prop descriptions

### PR DESCRIPTION
**Pull request by btraut**
Made a few additions/edits to Masonry docs in response to @rootvar's comments about lack of clarity on Masonry props.

### CODE REVIEW

**Comment by christianvuerings** [comment]
Deploy preview for *gestalt* ready!

Built with commit af8f083b6b1d50eb8b44ea86d84065979f135cc9

https://deploy-preview-417--gestalt.netlify.com

File: docs/src/Masonry.doc.js [hunk 1]
```diff
@@ -71,34 +71,38 @@ card(
         name: 'loadItems',
         type: '() => void',
         description:
-          'A callback when the user scrolls and you need to load more items into the grid.',
+          'A callback when the user scrolls and you need to load more items into the grid. Note that `scrollContainer` must be specified.',
```
Comment (rootvar) [inline docs/src/Masonry.doc.js]: This is great! I remember debugging `loadItems()` for a long time, not realizing that `scrollContainer` had to me specified.

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,7 @@
 ### Patch
 
 - Internal: Fixed a test that started flaking out with React 16.6 (#410)
+- Docs: Updated some Masonry props descriptions (#414)
```
Comment (pfarejowicz) [inline CHANGELOG.md]: the number is off here, should be #417

**Comment by pinterest-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  :microscope: Experimental data needed - <i>Changes were made to critical components. Do you have experimental data to back up your changes?</i>
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: CHANGELOG.md (hunk 2 above)
Comment (btraut) [reply CHANGELOG.md]: Fixed.

**Pull request merged**